plugins/kluster-capacity.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kluster-capacity
+spec:
+  version: v0.1.0
+  homepage: https://github.com/k-cloud-labs/kluster-capacity
+  shortDescription: Cluster capacity analysis tool support capacity estimation, cluster compression, scheduler simulation, fragmentation analysis and resource overview.
+  description: |
+    A simple CLI that support capacity estimation, cluster compression, scheduler simulation, fragmentation analysis and resource overview.
```

Comment:
Please wrap at 80 chars and explain the feature set as much as you can here. (It shouldn't be shorter than shortDescription.)

Reply:
done